script.js

```diff
@@ -1,4 +1,398 @@
+(4) If player: 21 Computer < 21 then player Wins. 
+(5) If player/ computer above 21, player/computer loses
+____________________________________________________________________________________________________________________
+*/
+
+/*********************************************************************************************************/
+// 1. Declaration of GLobal Variables.
+var deck;
+var cardDeck;
+var getCardDeck;
+var myOutputValue = "";
+var getDisplayOfDeck;
+var getDisplayOfShuffledDeck;
+
+var getShuffledDeck;
+
+var dealerTotal = 0;
+var playerTotal = 0;
+var dealerAceCount = 0;
+var playerAceCount = 0;
+var hiddenCard;
+var toStartGame;
+
+var toHit = true;
+
+var getHiddenCardValue;
+var getCardValue;
+var getAceCardValueForHidden;
+var getAceCardValue;
+var getDealerDrawCard;
+var getPlayerDrawCard;
+var getReduceAceValue;
+
+var clickHitButton;
+var clickStayButton;
+var clickStartGameButton;
+var clickResetButton;
+var blankOutcome;
+var finalPlayerPoints;
+var finalDealerPoints;
+var element;
+
+/*********************************************************************************************************/
+//2. Creation of a Deck.
+cardDeck = function () {
+  //Note the card file list as values.suits
+  let values = [
+    "A",
+    "2",
+    "3",
+    "4",
+    "5",
+    "6",
+    "7",
+    "8",
+    "9",
+    "10",
+    "J",
+    "Q",
+    "K",
+  ];
+  let suits = ["C", "D", "H", "S"];
+  // Curently the deck is empty.
+  deck = [];
+  // Proceed to populate the deck arrat using loops.
+  for (let i = 0; i < suits.length; i++) {
+    for (let j = 0; j < values.length; j++) {
+      deck.push(values[j] + "-" + suits[i]);
+    }
+  }
+  for (let i = 0; i < deck.length; i++) {
+    let cardImage = document.createElement("img");
+    let card = deck[i];
+    cardImage.src = "./card/" + card + ".png";
+    document.getElementById("originalCardDeck").append(cardImage);
+  }
+  console.log(deck);
+};
+
+console.log("Card Deck Array:");
+getCardDeck = cardDeck();
+
+/*********************************************************************************************************/
+// 3. Display Deck Array
+
+var displayDeck = function () {
+  myOutputValue = `Original Deck Array:` + `<br>` + `${deck}`;
+  return myOutputValue;
+};
+
+getDisplayOfDeck = displayDeck();
+
+/*********************************************************************************************************/
+// 4. Shuffle the Deck.
+
+var shuffledDeck = function () {
+  for (i = 0; i < deck.length; i++) {
+    // Scrambling the order so that the array can shuffled.
+    let j = Math.floor(Math.random() * deck.length);
+    let temp = deck[i];
+    deck[i] = deck[j];
+    deck[j] = temp;
+  }
+  for (let i = 0; i < deck.length; i++) {
+    let cardImage = document.createElement("img");
+    let card = deck[i];
+    cardImage.src = "./card/" + card + ".png";
+    document.getElementById("shuffledCardDeck").append(cardImage);
+  }
+};
+
+getShuffledDeck = shuffledDeck();
```

Comment:
the variable should be `shuffledDeck` and the function `shuffleDeck` or `getShuffledDeck`